src/dataSources/JSON.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+var Null = require('./Null');
```

Comment:
Let's get rid of Null altogether. We're not using it and we don't need it. Originally Steve had "undefined" dataSources set to null, but we're not doing that anymore. And with our plan to build pipelines to suit your app, it'll never happen.